svf/include/Graphs/CFBasicBlockG.h

```diff
@@ -493,6 +498,9 @@ class CFBasicBlockGraph : public GenericCFBasicBlockGTy
 class CFBasicBlockGBuilder
 {
 
+public:
+    typedef Map<const SVFBasicBlock*, std::vector<CFBasicBlockNode*>> SVFBBToCFBBNodes;
```

Comment:
Since the input of CFBasicBlock is ICFG,  we need to remove "SVFBasicBlock" from CFBasicBlockG. Clean up code and remove unnecessary data structures of SVF.